.github/workflows/coverage.yml

```diff
@@ -12,8 +12,8 @@ jobs:
           # needed for codecov
           fetch-depth: 0
       - name: Install tools
-        run: sudo apt-get install -qqy bats
+        run: sudo apt-get install -qqy bats criu
```

Comment:
Since we're not using restore functionality, I think we can leave it as is. The CRIU version provided by Ubuntu seems to work fine for dumping a process.

Reply:
JFYI, in GitHub CI criu is installed from [kubic:libcontainers:stable](https://build.opensuse.org/project/show/devel:kubic:libcontainers:stable) but I'm not sure if the packages are still [maintained](https://github.com/containers/podman.io_old/pull/584#issuecomment-1408335973):
https://github.com/actions/runner-images/blob/main/images/linux/scripts/installers/containers.sh

Reply:
In that case, is there a PPA or something I can add to use OBS instead? I'm not too familiar with Ubuntu package management.

Reply:
CRIU in GitHub Actions is not updated anymore as far as I know.

As long as we do not restore we can use the version from Ubuntu. That makes sense to me.

Reply:
>As long as we do not restore we can use the version from Ubuntu.

There are some changes to the *.proto files in newer versions of CRIU and the Go bindings are currently generated for the master branch. It would be good if we can test the functionality of checkpointctl with the latest version of CRIU.

>  is there a PPA or something I can add to use OBS instead?

We maintain both [OBS](https://build.opensuse.org/project/show/devel:tools:criu) and [PPA](https://launchpad.net/~criu) for CRIU.

Reply:
I've modified the workflow to use the PPA instead of the default package from Ubuntu's repository.